### PR TITLE
DRAFT: PWA Tutorial 1

### PR DESCRIPTION
**Pull request by estelle**
DRAFT!!!!!

This is a draft. Please do not edit typos, grammar, headers, links, etc. This is still very much in draft form, so whole sentences will be missing....  This is made public just to show progress and to get general feedback from the team working on PWAs and other interested parties.

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+You will be installing node.js, MAMP, or other local server to create a local secure connection. While the secure connection is a requirement, nodejs, MAMP, or any application other than a text editor and browser is not a requirement for creating a PWA.
+
+The steps include:
+
+- [Basic site](/en-US/Docs/Web/Progressive_web_apps/tutorials/intro/html)
+  - : We will create a website with a date picker and an installation button, using HTML, CSS, and JavaScript.
+- [Icon and appearance](/en-US/Docs/Web/Progressive_web_apps/tutorials/intro/manifest)
+  - : A PWA requires a manifest is a JSON file that describes the name, icon, description, and other definitions of how the application works within the operating system on which the PWA is installed. We will create a manifest file that defines the appearance of the application when installed, including which icons should be used depending on the user's device, and parameters for the application viewport.
```
Comment (dawei-wang) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md:60]:
```suggestion
  - : A PWA requires a manifest which is a JSON file that describes the name, icon, description, and other definitions of how the application works within the operating system on which the PWA is installed. We will create a manifest file that defines the appearance of the application when installed, including which icons should be used depending on the user's device, and parameters for the application viewport.
```

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+- [Secure connection](/en-US/Docs/Web/Progressive_web_apps/tutorials/intro/secure)
+  - : While no frameworks are needed to create a PWA, we will link to options for creating a secure localhost connection so you can test your application and install it.
+- [Service worker](/en-US/Docs/Web/Progressive_web_apps/tutorials/intro/service_worker)
+  - : The service worker enables the application to work offline. With the secure connection in the previous step, On the very first visit to a web page that installs a new service worker, the initial visit to a page provides its baseline functionality while the service worker downloads. After a service worker is installed and activated, it controls the page to offer improved reliability and speed.
```
Comment (dawei-wang) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md:64]:
```suggestion
  - : The service worker enables the application to work offline. With the secure connection in the previous step, the initial visit to a page provides its baseline functionality while the service worker downloads. After a service worker is installed and activated, it controls the page to offer improved reliability and speed.
```

Comment (dawei-wang) [reply files/en-us/web/progressive_web_apps/tutorials/intro/index.md:64]: Seems like a redundant sentence.

**Review by dawei-wang — COMMENTED** [review]
A few typos

**Comment by schalkneethling** [comment]
Hey, @dawei-wang - Thank you for your eagerness to review and provide feedback here. While a pull request is still in draft, it is a signal from the author that this is not ready for review. As a result, you might be offering feedback on something very likely to change completely. I would therefore ask that you wait until the author has marked the pull request as ready for review. Thanks!

**Pull request closed without merging**